Description: Remove dependency on criterion
 This is needed only for the benchmark and complicates packaging.
Author: Johann Felix Soden <johfel@debian.org>
Forwarded: not-needed
Last-Updated: 2023-07-16
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -30,13 +30,8 @@
 name = "integration"
 required-features = ["std"]
 
-[[bench]]
-name = "benchmark"
-harness = false
 [dependencies.unicode-width]
 version = "0.1.8"
-[dev-dependencies.criterion]
-version = "0.3.3"
 
 [features]
 default = ["std"]
--- a/benches/benchmark.rs
+++ b/benches/benchmark.rs
@@ -1,3 +1,4 @@
+/*
 use criterion::{criterion_group, criterion_main, Criterion, Throughput, BenchmarkId};
 
 use unicode_truncate::UnicodeTruncateStr;
@@ -32,3 +33,4 @@
 
 criterion_group!(benches, criterion_benchmark);
 criterion_main!(benches);
+*/
